app/helpers/downloads_helper.rb

```diff
@@ -0,0 +1,78 @@
+module DownloadsHelper
+  def download_options options_array, domain, token
```

Comment:
I feel like this could be refactored into a  `case when` statement, or at least it could be condensed into a smaller piece of code especially with the outcome being that a hash is being pushed to the `download_options` array

Reply:
@stacytalbot That's done now